wg-automl/charter.md

```diff
@@ -19,7 +19,7 @@ The WG covers researching, developing and operating various targets of ML automa
 #### Cross-cutting and Externally Facing Processes
 
 - Coordinating with Training WG to make sure that all distributed training jobs can be used in AutoML experiments.
```

Comment:
> Coordinating with Training WG to make sure that all distributed training jobs can be used in AutoML experiments.

Why is this in scope for this WG?

Reply:
I think autoML covers this part.  https://github.com/kubeflow/community/blame/master/wg-automl/charter.md#L21
It can be removed here.

Reply:
> Coordinating with Training WG to make sure that all distributed training jobs can be used in AutoML experiments.

This line is not related to this PR, unless I am missing something.